src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1720,25 +1741,26 @@ void Partitioner::optimize(const std::string& func_name) {
             for (auto&& funcall : func_group.refs) {
                 auto new_elem_type = params_to_gather.pnew->get_element_type();
                 auto new_shape = params_to_gather.pnew->get_shape();
-                funcall.get()._closure.push_back(ov::Tensor(new_elem_type, new_shape));
+                funcall.get()._lazy_closure.push_back(
+                    LazyTensor(TransformType::THIS, ov::Tensor(new_elem_type, new_shape)));
```

Comment:
This tensor is not a weight to be shared and cached!!!
This is a placeholder tensor for host-side gather.

Probably, it can be stored in the `_cloure` as it is - so it will be the only one physical buffer there at this point.

What host_site_gather is ON, we populate this tensor on host. So it is more like a runtime parameter rather than immutable const.

Reply:
Done